config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigquerydatatransferconfigs.bigquerydatatransfer.cnrm.cloud.google.com.yaml

```diff
@@ -123,128 +146,87 @@ spec:
+
+                   NOTE: The minimum interval time between recurring transfers depends on the
+                   data source; refer to the documentation for your data source.
                 type: string
               scheduleOptions:
                 description: Options customizing the data transfer schedule.
                 properties:
                   disableAutoScheduling:
-                    description: |-
-                      If true, automatic scheduling of data transfer runs for this
-                      configuration will be disabled. The runs can be started on ad-hoc
-                      basis using transferConfigs.startManualRuns API. When automatic
+                    description: If true, automatic scheduling of data transfer runs
+                      for this configuration will be disabled. The runs can be started
+                      on ad-hoc basis using StartManualTransferRuns API. When automatic
                       scheduling is disabled, the TransferConfig.schedule field will
                       be ignored.
                     type: boolean
                   endTime:
-                    description: |-
-                      Defines time to stop scheduling transfer runs. A transfer run cannot be
-                      scheduled at or after the end time. The end time can be changed at any
-                      moment. The time when a data transfer can be triggered manually is not
-                      limited by this option.
+                    description: Defines time to stop scheduling transfer runs. A
+                      transfer run cannot be scheduled at or after the end time. The
+                      end time can be changed at any moment. The time when a data
+                      transfer can be triggered manually is not limited by this option.
                     type: string
                   startTime:
-                    description: |-
-                      Specifies time to start scheduling transfer runs. The first run will be
-                      scheduled at or after the start time according to a recurrence pattern
-                      defined in the schedule string. The start time can be changed at any
-                      moment. The time when a data transfer can be triggered manually is not
-                      limited by this option.
+                    description: Specifies time to start scheduling transfer runs.
+                      The first run will be scheduled at or after the start time according
+                      to a recurrence pattern defined in the schedule string. The
+                      start time can be changed at any moment. The time when a data
+                      transfer can be triggered manually is not limited by this option.
                     type: string
                 type: object
-              sensitiveParams:
```

Comment:
This field only exists in Terraform.
https://registry.terraform.io/providers/hashicorp/google-beta/latest/docs/resources/bigquery_data_transfer_config#sensitive_params